src/utils/constants.ts

```diff
@@ -8,7 +8,7 @@ export const DEFAULT_CHAIN_ID =
 export const PUBLIC_URL = process.env.PUBLIC_URL
 export const TX_SERVICE_VERSION = '1'
 export const INTERCOM_ID = IS_PRODUCTION ? process.env.REACT_APP_INTERCOM_ID : 'plssl1fl'
-export const BEAMER_ID = IS_PRODUCTION ? process.env.REACT_APP_BEAMER_ID || '' : 'kqHQMTfH40865'
+export const BEAMER_ID = IS_PRODUCTION ? process.env.REACT_APP_BEAMER_ID || '' : 'ehlRMhQi41258'
```

Comment:
To not be forgotten later, I suggest we include the GitHub secret for `BEAMER_ID`

Reply:
It's on you right @katspaugh ?

Reply:
Added, you can remove the fallback.

Reply:
Should we not keep it as a fallback like we do with `INTERCOM_ID`? The `BEAMER_ID` will be publicly accessible via `window.beamer_config` in any case.